packages/core-api/src/lib/AuthSessionManager/types.ts

```diff
@@ -36,6 +36,13 @@ export type SessionManager<T> = {
   sessionState$(): Observable<SessionState>;
 };
 
+// FIXME do we need this?
+export type SamlAuthSessionManager<T> = {
```

Comment:
I could be wrong but the type is similar to SessionManager<T> and don't see a specific need for this.